src/com/hms/flexyosisoftconnector/OSIsoftConfig.java

```diff
@@ -162,11 +165,14 @@ public static void initConfig(String configFile) throws JSONException, IOExcepti
       System.exit(1);
     }
 
+    boolean res;
     if (appConfig.has("LoggingLevel")) {
-      boolean res = Logger.SET_LOG_LEVEL(appConfig.getInt("LoggingLevel"));
-      if (!res) {
-        Logger.LOG_SERIOUS("Invalid logging level specified");
-      }
+      res = Logger.SET_LOG_LEVEL(appConfig.getInt("LoggingLevel"));
+    } else {
+      res = Logger.SET_LOG_LEVEL(LOGGING_LEVEL_DEFAULT);

```

Comment:
Should there be a message here saying that there is no logging level in the config file?

Reply:
added